README.md

```diff
@@ -63,6 +63,18 @@ With these pieces of information on hand, you’re ready to have your end users
 https://dashboard.checkr.com/oauth/authorize/:client_id?scope=read_write
 ```
 
+You can also specifically redirect users to the /signin or /signup endpoint specifically depending on whether you know they have already set up an account with Checkr. The URLs for both options are as follows:
+
+**Sign in**
+```
+https://partners.checkr.com/authorize/:client_id/signin?scope=read_write&ref=dashboard
```

Comment:
You should remove `&ref=dashboard`. That is added by dashboard so we can see how many people are still redirecting to `dashboard.checkr.com` vs directly to `partners.checkr.com`